### ISSUE TITLE
We could do we an automated test-suite for some of the longer setups

### ISSUE THREAD

**Issue by FilWisher** [FilWisher/dear-content#6]
Setting up a circular loop to test how many echoes are allowed is a very long process

**Comment by rjmk** [comment]
Yeah that was pretty boring. I get 5 with 4 loopers and one head, by the way

**Comment by FilWisher** [comment]
Any thoughts on how to approach automating this?

**Comment by rjmk** [comment]
One could hack a shell / go script to run the programs with given arguments?

In a more standard way, one could copy some of the setup from main into a TestMain and pass an alternative (chan?) to stdio to give commands
